File: lib/src/main/java/com/darklycoder/wifitool/lib/interfaces/ConnectWiFiActionListener.java
package com.darklycoder.wifitool.lib.interfaces;

import android.net.wifi.WifiConfiguration;

import com.darklycoder.wifitool.lib.type.Types;

public interface ConnectWiFiActionListener extends IActionListener {

    void onCreateConfig(WifiConfiguration configuration);

    void onResult(@Types.ConnectResultType int type);

}

File: lib/src/main/java/com/darklycoder/wifitool/lib/interfaces/ConnectWiFiActionListener.kt
package com.darklycoder.wifitool.lib.interfaces

import android.net.wifi.WifiConfiguration

import com.darklycoder.wifitool.lib.type.Types

interface ConnectWiFiActionListener : IActionListener {

    fun onCreateConfig(configuration: WifiConfiguration)

    fun onResult(@Types.ConnectResultType type: Int)

}
